From 5bd3d7194858264e9917137ad5ef2472522d0ae9 Mon Sep 17 00:00:00 2001
From: Thorsten Horstmann <thorsten.horstmann@fkie.fraunhofer.de>
Date: Thu, 12 Jun 2025 18:01:19 +0200
Subject: [PATCH 1/1] Thin UHD compilation - Part II

---
 cmake_targets/tools/build_helper | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake_targets/tools/build_helper b/cmake_targets/tools/build_helper
index c45cb636a1..ce313989a8 100755
--- a/cmake_targets/tools/build_helper
+++ b/cmake_targets/tools/build_helper
@@ -290,7 +290,7 @@ install_usrp_uhd_driver_from_source(){
     git diff
     mkdir -p host/build
     cd host/build || true
-    $CMAKE ../ -GNinja -DENABLE_TESTS=OFF -DENABLE_EXAMPLES=OFF
+    $CMAKE ../ -GNinja -DENABLE_TESTS=OFF -DENABLE_EXAMPLES=OFF -DENABLE_MANUAL=OFF -DENABLE_MAN_PAGES=OFF -DENABLE_DOXYGEN=OFF 
     echo "Compiling UHD"
     ninja
     $SUDO ninja install
-- 
2.43.0

